web/styles/right_sidebar.css

```diff
@@ -13,11 +13,37 @@ $user_status_emoji_width: 24px;
     overflow: auto;
 }
 
-#buddy-list-users-matching-view {
-    max-width: 95%;
+.toggle-narrow-users,
+.toggle-other-users {
+    width: 7px;
+}
+
+#buddy-list-users-matching-view-container,
+#buddy-list-other-users-container {
+    margin-bottom: 10px;
+
+    &.no-display {
+        display: none;
+    }
+
+    .view-all-subscribers-link,
+    .view-all-users-link {
+        margin-left: 15px;
+    }
+}
+
+#buddy-list-users-matching-view,
+#buddy-list-other-users {
+    max-width: 90%;
```

Comment:
Should this just be a `width` value? `max-width` set to a percentage is usually the road to frustration, and doesn't differ appreciably from plain old `width`

Reply:
It's been too long since I wrote this to remember 😅 I'll take your word for it and switch to `width` though, looks okay for now